docs/usage/configuration-options.md

```diff
@@ -1248,6 +1248,27 @@ This way the secret can be used in both the `org1` and `org2` organizations.
 
 For more information on how to use secrets for private packages, read [Private package support](./getting-started/private-packages.md).
 
+## env
+
+This option allows users to specify explicit environment variables and their corresponding values.
+
+<!-- prettier-ignore -->
+!!! note
+    This option is only available to users if the bot administrator has configured an [`allowedEnv`](./self-hosted-configuration.md#allowedEnv) list permitting the env names you wish to use.
+
+Behavior:
+
+- This option is only applicable to when executing package manager commands, specifically within the `updateArtifacts()` function
+- Values defined in the env configuration take precedence over corresponding environment variables, including those from `customEnvVariables` and `process.env`
```

Comment:
```suggestion
- Values set in the env configuration override corresponding environment variables, including those from `customEnvVariables` and `process.env`
```

`Take precedence over` is difficult English to me. I think a word like `override` is easier.